api/licenses/views.py

```diff
@@ -58,7 +58,7 @@ class LicenseList(JSONAPIBaseView, generics.ListAPIView, ListFilterMixin):
     ordering = ('name', )  # default ordering
 
     def get_default_queryset(self):
-        return NodeLicense.objects.all()
+        return NodeLicense.objects.exclude(license_id='CCBYNCND').exclude(license_id='CCBYSA40')
```

Comment:
It's not really clear why these are being excluded--can we have way for the code to more clearly separate project licenses from preprint-only licenses?

One idea might be to use custom managers (see https://docs.djangoproject.com/en/1.11/topics/db/managers/#modifying-a-manager-s-initial-queryset) to automatically filter:

```python
NodeLicense.objects.project_licenses.all()  # excludes CCBYNCND and CCBYSA40
NodeLicense.objects.preprint_licenses.all()
```

Reply:
that's a good idea

Reply:
will wait to change this @sloria until there's consensus that this is the route we're going. This WIP PR was just in response to being asked if this was possible.

Reply:
Change made @sloria